ext/datasource/hotspot_rule_converter.go

```diff
@@ -0,0 +1,98 @@
+package datasource
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/alibaba/sentinel-golang/core/hotspot"
+	"github.com/alibaba/sentinel-golang/logging"
+	"github.com/pkg/errors"
+)
+
+type HotspotRule struct {
+	hotspot.Rule
+	ExtDsSpecificItems []SpecificValue `json:"extDsSpecificItems"`
+}
+
+// ParamKind represents the Param kind.
+type ParamKind int
+
+const (
+	KindInt ParamKind = iota
+	KindString
+	KindBool
+	KindFloat64
+	KindSum
+)
+
+func (t ParamKind) String() string {
+	switch t {
+	case KindInt:
+		return "KindInt"
+	case KindString:
+		return "KindString"
+	case KindBool:
+		return "KindBool"
+	case KindFloat64:
+		return "KindFloat64"
+	default:
+		return "Undefined"
+	}
+}
+
+// SpecificValue indicates the specific param, contain the supported param kind and concrete value.
+type SpecificValue struct {
+	ValKind   ParamKind `json:"valKind"`
```

Comment:
Can you verify valkind using https://github.com/go-playground/validator

Reply:
The idea is well, From my perspective, I think it does more harm than good.
Sentinel, as a base component, is not a good choice for introducing too many dependent packages. We should keep package dependencies pure.